plugins/modules/idrac_certificates.py

```diff
@@ -399,6 +421,26 @@ def get_cert_url(actions, op, certype, res_id):
     return dynurl
 
 
+def upload_ssl_key(module, idrac, actions, ssl_key, res_id):
+    if not (os.path.exists(ssl_key) or os.path.isdir(ssl_key)):
```

Comment:
if not os.path.exists(ssl_key) or os.path.isdir(ssl_key):

Reply:
Done